torch/utils/bottleneck/__main__.py

```diff
@@ -223,9 +223,10 @@ def main():
     # if their execution times are very different.
     cuda_prof_exec_time = cpu_time_total(autograd_prof_cuda)
     cpu_prof_exec_time = cpu_time_total(autograd_prof_cpu)
-    pct_diff = cuda_prof_exec_time - cpu_prof_exec_time / cuda_prof_exec_time
-    if abs(pct_diff) > 0.05:
-        print_autograd_prof_summary(autograd_prof_cpu, 'CPU', autograd_prof_sortby, autograd_prof_topk)
+    if cuda_prof_exec_time > 0.0:
```

Comment:
Actually in which case can this be 0? I don't fully understand. You function *will take some time*, so it should be ok.

Reply:
Actually, I was recurrent a case related to performance. I remember that has little cuda operation.
When I try to use bottleneck to profile it, this error occurs, so I start to fix it.

Reply:
@egg-west what do you mean by lack of bracket?

I'm able to reproduce this with the following things:
- empty script -- records nothing on the profiler
- `import torch; torch.empty([])` -- records nothing on the profiler

In general the CUDA time should always be nonzero because we're recording a CUDA event, running the function, and then recording another CUDA event, unless that function does nothing (and even then, python overhead adds some time). This is true even if the program doesn't use CUDA. The divide by zero error happens only when the profiler records nothing.

Reply:
'lack of bracket' cause the priority error in `pct_diff = (cuda_prof_exec_time - cpu_prof_exec_time) / cuda_prof_exec_time`
I will test it again later.